doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -107,7 +108,9 @@ The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from
 
```

Comment:
You need to update the SHA in line 107 whenever Matter SDK is updated.
See for example: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/releases_and_maturity/releases/release-notes-2.2.0.html#matter-fork

Reply:
Thanks for the reminder. I will update it after merging the PR in SDK.

Reply:
Wait, @greg-fer we need to update the SHA after the alignment of sdk_connectedhomeip to a specific version of the Matter Upstream SDK. In this PR the `v1.1.0.1` version of the Matter SDK is still correct.

Reply:
Good to know, and sorry for the hassle. :)